stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java

```diff
@@ -61,6 +61,7 @@
 @Path(FileResource.SERVICE_PATH)
 public class MockFileResource implements FileResource {
     @Override
+    @Deprecated
```

Comment:
There's already JavaDoc in the parent class `FileResource`. I'm not sure if we should update all method that implements `FileResource`.